src/blueprint/server.rs

```diff
@@ -42,13 +40,12 @@ impl Server {
   }
 }
 
-impl TryFrom<crate::config::Server> for Server {
+impl TryFrom<crate::config::Config> for Server {
```

Comment:
shouldn't Server be created from Server?

Reply:
oh! yes we can after move the upstream out of server. will do.

Reply:
done